Makefile

```diff
@@ -41,10 +41,20 @@ create-db:
 
 apply-migrations: ensure-venv
 	@echo "--> Applying migrations"
-	sentry upgrade
+	sentry upgrade --noinput
```

Comment:
This removes the prompt to create a Django user. `make create-user` deals with it later.